packages/web-config-server/.env.example

```diff
@@ -0,0 +1,35 @@
+AWS_ACCESS_KEY_ID=
+AWS_ACCESS_KEY_SECRET=
+CONFIG_SERVER_BASE_URL=
+COOKIE_SECRET=
+DHIS_CLIENT_ID=
+DHIS_CLIENT_SECRET=
+DHIS_PASSWORD=
+DHIS_USERNAME=
+EXPORT_COOKIE_URL=
+EXPORT_URL=
+LAST_USER_SESSION_COOKIE_SECRET=
+LOG_FILE_NAME=
+ORGANISATION_UNIT_CACHE_PERIOD=
+PORT=
+SITE_EMAIL_ADDRESS=
+SMTP_HOST=
+SMTP_PASSWORD=
+SMTP_USER=
+TUPAIA_APP_SERVER_AUTH=
+TUPAIA_APP_SERVER_URL=
+USER_SESSION_COOKIE_SECRET=
+IS_PRODUCTION_ENVIRONMENT=
+VANUATU_DHIS_OAUTH_CLIENT_NAME=
+VANUATU_DHIS_OAUTH_CLIENT_SECRET=
+VANUATU_DHIS_OAUTH_PASSWORD=
+VANUATU_DHIS_OAUTH_USERNAME=
```

Comment:
These can be deleted, we rolled back our integration with Vanuatu's DHIS2 server